### PR TITLE
Add doc for context displayaName

### PR DESCRIPTION
**Pull request by M-Izadmehr**
This PR adds a description of `Context.displayName` which is used to show a specific name for the React DevTools.
Also solved [this Issue](https://github.com/facebook/react/issues/16851)

### CODE REVIEW

File: content/docs/context.md [hunk 1]
```diff
@@ -196,6 +197,20 @@ Requires a [function as a child](/docs/render-props.html#using-props-other-than-
 > 
 > For more information about the 'function as a child' pattern, see [render props](/docs/render-props.html).
 
+### `Context.displayName` {#contextdisplayName}
```
Comment (eps1lon) [inline content/docs/context.md]: This doesn't get parsed properly it seems. Maybe only lowercase is allowed?

Comment (M-Izadmehr) [reply content/docs/context.md]: Yeah it seems that all links are in lowercase, I changed it to lowercase and pushed it.

File: content/docs/context.md [hunk 2]
```diff
@@ -196,6 +197,20 @@ Requires a [function as a child](/docs/render-props.html#using-props-other-than-
...
+
+Context object accepts a `displayName` string property. React DevTools uses this string to determine what to display for the context.
+
+For example, the following component will appear as MyContext in the DevTools:
+
+```js{2}
+const MyContext = React.createContext(/* some value */);
+MyContext.displayName = 'MyDisplayName';
+
+<MyContext.Provider> // "MyContext.Provider" in DevTools
```
Comment (alexkrolick) [inline content/docs/context.md]: shouldn't this be `// "MyDisplayName.Provider" in DevTools`?

Comment (M-Izadmehr) [reply content/docs/context.md]: Done:)

**Comment by alexkrolick** [comment]
thank you!

**Pull request merged**